ocs_ci/ocs/cluster.py

```diff
@@ -912,6 +912,61 @@ def get_osd_utilization():
     return osd_filled
 
 
+def get_ceph_df_detail():
+    """
+    Get ceph osd df detail
+    :return:
+        dict returned from cmd
+    """
+    ceph_cmd="ceph df detail"
+    ct_pod = pod.get_ceph_tools_pod()
+    output = ct_pod.exec_ceph_cmd(ceph_cmd=ceph_cmd)
+    return output
+
+
+def validate_replica_data(pool_name, replica):
+    """
+    Check if data is replica 2 or 3
+    :param replica: (int) size of the replica(2,3)
+    :param pool_name: (string) name of the pool to check replica
+    :return: true if replica is verified
+    """
+    output = get_ceph_df_detail()
+    for pool in output.get('pools'):
+        if pool.get('name') == pool_name:
+            logger.info(f"{pool_name}")
+            stored = pool.get('stats').get('stored')
+            byte_used = pool.get('stats').get('bytes_used')
+            compress_bytes_used = pool.get('stats').get('compress_bytes_used')
+            compress_under_bytes = pool.get('stats').get('compress_under_bytes')
+            byte_used = byte_used + compress_under_bytes - compress_bytes_used
+            store_ratio = byte_used / stored
+            assert (replica+0.2) > store_ratio > (replica-0.2), f"pool {pool_name} do not meet rep {replica} size"
+            return True
```

Comment:
since there is no return False in this function, I would remove the `return True` statement and change this function to not return anything

Reply:
Made it return true or false because of Martin comment to have the assert in the testcase itself